src/Microsoft.IdentityModel.Tokens/Exceptions/SecurityTokenInvalidSigningKeyException.cs

```diff
@@ -73,6 +73,8 @@ public SecurityTokenInvalidSigningKeyException(string message, Exception inner)
         protected SecurityTokenInvalidSigningKeyException(SerializationInfo info, StreamingContext context)
             : base(info, context)
         {
+            //TODO - investigate if SigningKey type should be serialized
```

Comment:
> //TODO - investigate if SigningKey type should be serialized [](start = 11, length = 61)

The answer to this should be why we are doing this work for serialization. Are we transferring these over the wire?

Reply:
I don't think so. I think this would be used by users for writing exceptions to files.

Reply:
I agree we should remove that TODO.